### PR TITLE
Better material ui

### PR DESCRIPTION
**Pull request by sebastianovide**

### CODE REVIEW

**Comment by sebastianovide** [comment]
for #11 and #94

File: src/components/PhotoPage.js [hunk 1]
```diff
@@ -112,30 +120,31 @@ class PhotoPage extends Component {
...
   componentDidMount() {
-    this.loadImage();
-
     window.gtag('event', 'page_view', {
       'event_category': 'view',
       'event_label': 'PhotoPage'
     });
   }
 
+  componentDidUpdate = (nextProps) => {
```
Comment (kdalkafoukis) [inline src/components/PhotoPage.js]: nextProps should be renamed to prevProps
https://reactjs.org/docs/react-component.html#componentdidupdate

File: src/App.js [hunk 2]
```diff
@@ -94,36 +127,103 @@ class App extends Component {
...
-               file={this.state.file}
-               location={this.state.location}
-               online={this.state.online}
-            />}
-          />
-          <Route path='/map' render={(props) => <Map {...props} location={this.state.location} />}/>
-          <Route path='/signedin' component={SignedinPage} />
-        </Switch>
+        <Login
+          open={this.state.loginLogoutDialogOpen && this.state.isSignedIn !== undefined && !this.state.isSignedIn}
```
Comment (kdalkafoukis) [inline src/App.js]: Do we need to check this.state.isSignedIn twice?

File: src/App.js [hunk 3]
```diff
@@ -1,34 +1,67 @@
...
     super(props);
+
     this.state = {
       file: null,
       location: {},
       isSignedIn: null,
       photosToModerate: [],
-      online:false
+      online: false,
+      pageTitle: "Map",
```
Comment (kdalkafoukis) [inline src/App.js]: Do we use/change somewhere pageTitle to put it in state?

**Pull request merged**